src/test/components/__snapshots__/ListOfPublishedProfiles.test.js.snap

```diff
@@ -672,7 +672,7 @@ exports[`ListOfPublishedProfiles still displays the link to all recordings when
     <a
       href="/uploaded-recordings/"
     >
-      Manage these recordings
+      Manage this recording
```

Comment:
I don't understand why is this happening. So apparently the text is not changing depending on the length as it should. But in the markup, the jsx is written using conditional. So I had to add only one string for that and make it changeable depending its length in the FTL file. So what to do here? @canova

Reply:
As we discussed, I believe this happens because we're not handling the FTL file in this test. In the future I think we'll want to hook the en-US file to all tests, but this is out of scope here. So let's say this change is OK, provided you double check this works as expected in the real product :-)